pkgs/development/libraries/science/astronomy/indilib/default.nix

```diff
@@ -38,19 +40,30 @@ stdenv.mkDerivation rec {
     libjpeg
     gsl
     fftw
+    libev
   ];
 
   cmakeFlags = [
     "-DCMAKE_INSTALL_LIBDIR=lib"
     "-DUDEVRULES_INSTALL_DIR=lib/udev/rules.d"
   ];
+  postPatch = ''
+    for f in drivers/*/*.rules;
+    do
+      substituteInPlace $f \
+        --replace "/lib/udev/rules.d" "lib/udev/rules.d" \
+        --replace "/etc/udev/rules.d" "lib/udev/rules.d" \
+        --replace "/lib/firmware" "lib/firmware" \
+        --replace "/bin/sh" "${bash}/bin/sh"
```

Comment:
This is already set up by NixOS (see [here](https://github.com/NixOS/nixpkgs/blob/b2544f1301520be456fc33d296e6c9a5f6126391/nixos/modules/config/shells-environment.nix#L137)) -- was this required? (It shouldn't be.)

Reply:
Yep, for the same reason as fxload; the udev module is pretty naive. See https://github.com/NixOS/nixpkgs/blob/master/nixos/modules/services/hardware/udev.nix#L50